### PR TITLE
Add GPS and Newtonzz Step Counter

### PR DESCRIPTION
**Pull request by Hkarimi561**

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -49,6 +142,18 @@ RegisterNUICallback('setThirstWarning', function(data)
     QBCore.Functions.Notify('Fitbit: Thirst warning set to '..thirstValue..'%')
 end)
 
+
+RegisterNUICallback('setStepCount', function(data)
+
+    PRPCore.Functions.Notify('Fitbit: Step counter reset!')
```
Comment (Choppa25) [inline client/main.lua:148]: This should be

`QBCore.Functions.Notify('Fitbit: Step counter reset!')`

File: html/app.js [hunk 2]
```diff
@@ -1,66 +1,99 @@
 var openedApp = ".main-screen"
 
 qbFitbit = {}
+var steps = "233.5k"
```
Comment (Choppa25) [inline html/app.js:5]: Add this get rid of **Uncaught ReferenceError: curentapp is not defined**

`var currentapp `

File: client/main.lua (hunk 1 above)
Comment (Hkarimi561) [reply client/main.lua]: i fix this

File: html/app.js (hunk 2 above)
Comment (Hkarimi561) [reply html/app.js:5]: i fix this

**Comment by darknight2590** [comment]
Thank you, can you add a timer I think it is also in the original version I tried before
This update will open a lot to develop the watch and add new features to it

**Comment by Hkarimi561** [comment]
@darknight2590 you mean count down timer and something like that

**Comment by Choppa25** [comment]
Correct me if I'm wrong, but the directions app is pretty much pointless as it is. You can't keep the watch open and move/drive at the same time. Maybe something could be done about that.

**Review by evanillaa — APPROVED** [review]

**Comment by vosscat** [comment]
How im gonna be able to remove the `key disable` while have the fitbit on and also able to move?

**Pull request closed without merging**